tests/ui-fulldeps/lexer/unicode-version.rs

```diff
@@ -0,0 +1,21 @@
+// This test is used to validate which version of Unicode is used for parsing
+// identifiers. If the Unicode version changes, it should also be updated in
+// the reference at
+// https://github.com/rust-lang/reference/blob/HEAD/src/identifiers.md.
+
+//@ run-pass
+//@ check-run-results
+//@ ignore-cross-compile
+//@ reference: ident.unicode
+//@ reference: ident.normalization
+
+#![feature(rustc_private)]
+
+extern crate rustc_driver;
+extern crate rustc_lexer;
+extern crate rustc_parse;
+
+fn main() {
```

Comment:
Suggestion: since we're going to capture run stdout anyway, can you add a `println!` saying something like

```rs
println!("Checking if Unicode version changed.");
println!("If the Unicode version changes are intentional, it should also be updated in the reference at https://github.com/rust-lang/reference/blob/HEAD/src/identifiers.md.");
```

to make test failures less surprising and easier to figure out what to do about it?

Reply:
Done.